interview-questions.md

```diff
@@ -8,62 +8,69 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: The parameter is the placeholder inside a function. When the argument is passed, the argument is connected to the parameter and we can then create actions on the parameter within the function.  
 
-  Researched answer:
+  Researched answer:A parameter is the variable that is listed in the parentheses of the function, and when something is passed through the console.log (which is the argument) it is sent through that function.
 
 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
+  Your answer: The required parameters are the ones that need to be pushed from the argument. When an argument is passed, the map() function will alter the original array. 
 
-  Researched answer:
+  Researched answer: the array method called map(), calls a specified function for every array element and then creates a new array with that. This map() method does not change anything to the original array but creates a new array. We use this if we want to execute a different function to an array or an element within that array. 
```

Comment:
The predefined parameters in this case is referencing what params you would set when writing .map which is currentValue, index and array.  `array.map((value, index) => value`